changes/ocd-3803.sql

```diff
@@ -0,0 +1,7 @@
+update openchpl.optional_standard
+set description = 'CCDS: SNOMED CT® U.S. Edition, September 2019 Release'
```

Comment:
Does the symbol after the "CT" show up correctly in all of the various places? XML, HTML, email (if these are in any emails), etc.?

Reply:
It works in the HTML and here is in the XML.  Looks good in my viewer.  Not sure if it appears in any emails.
![image](https://user-images.githubusercontent.com/24817202/141529805-5041174d-ccbb-4843-889d-3deee000dd14.png)